ddtrace/contrib/molten/__init__.py

```diff
@@ -0,0 +1,16 @@
+"""
+To trace the molten web framework, install the trace middleware::
+
+
+To enable distributed tracing when using autopatching, set the
+``DATADOG_FALCON_DISTRIBUTED_TRACING`` environment variable to ``True``.
```

Comment:
TODO: We need to update these docs with actual usage and to not say "Falcon" :)

Reply:
DONE